include/seastar/core/sstring.hh

```diff
@@ -804,3 +805,7 @@ std::ostream& operator<<(std::ostream& os, const std::unordered_map<Key, T, Hash
     return os;
 }
 }
+
+#if FMT_VERSION >= 90000
+template <typename T> struct fmt::formatter<std::vector<T>> : fmt::ostream_formatter {};
+#endif
```

Comment:
I think this place also deserves `fmt::formatter<std::unordered_map<Key, T, Hash, KeyEqual, Allocator>>`

Reply:
added
```c++
template <typename... Args> struct fmt::formatter<std::unordered_map<Args...>> : fmt::ostream_formatter {};
```
instead.